dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Controls/ExciterAvrPssOel.mo

```diff
@@ -0,0 +1,155 @@
+within Dynawo.Examples.Nordic.Components.GeneratorWithControl.Controls;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model ExciterAvrPssOel
+
+  import Modelica;
+  import Dynawo.Connectors;
+
+  parameter Real G;
+  parameter Real Ta;
+  parameter Real Tb;
+  parameter Real L2;
+  parameter Real Kp;
+  parameter Real Tw;
+  parameter Real T1;
+  parameter Real T2;
+  parameter Real L1;
+  parameter Real r;
+  parameter Real f;
+  parameter Real IfLimPu;
+
+  Modelica.Blocks.Interfaces.RealInput UStatorPu(start = UStator0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-182, -44}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput ifPu(start = If0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-182, 56}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, 60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealOutput efdPu(start = Efd0Pu) annotation(
+    Placement(visible = true, transformation(origin = {228, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Connectors.ImPin efdPuPin(value(start = Efd0Pu));
+  Modelica.Blocks.Math.Min dU_OEL_min annotation(
```

Comment:
Can you please add a line after the inputs/ouputs/connectors and the blocks ? Thanks
Also, could you add a comment with units and pu base for each input, output, connector ? Thanks

Reply:
resolved, see [e1000c7](https://github.com/dynawo/dynawo/pull/2394/commits/e1000c7adff25e271a2e2214bb7197d005b00633)